targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L476xG/TARGET_RHOMBIO_L476DMW1K/PinNames.h

```diff
@@ -0,0 +1,375 @@
+/* mbed Microcontroller Library
+
+* SPDX-License-Identifier: Apache-2.0
```

Comment:
SPDX is wrong in some files, the below is BSD-style license but SPDX states Apache 2.0? Please fix